### ISSUE TITLE
Error when calling browserSupportsWebAuthnAutofill() if navigator.credentials is not defined

### ISSUE THREAD

**Issue by sameh-amwal** [MasterKale/SimpleWebAuthn#295]
I suggest changing the code to be more defensive
```
    if (navigator?.credentials?.conditionalMediationSupported) {
```

<img width="940" alt="Screenshot 2022-10-27 at 9 14 01 PM" src="https://user-images.githubusercontent.com/100665288/198378686-8ea03d00-fd12-4609-8f72-4628723bcae6.png">

**Comment by MasterKale** [comment]
Good callout, thanks @sameh-amwal. That line is actually supporting old conditional UI logic in Chrome Canary. As of Chrome Canary 109 the `isConditionalMediationAvailable()` flag is the correct one, so I can probably finally remove the lines that're causing the issue here:

https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/browserSupportsWebAuthnAutofill.ts#L11-L13

**Comment by sameh-amwal** [comment]
> Good callout, thanks @sameh-amwal. That line is actually supporting old conditional UI logic in Chrome Canary. As of Chrome Canary 109 the `isConditionalMediationAvailable()` flag is the correct one, so I can probably finally remove the lines that're causing the issue here:
> 
> https://github.com/MasterKale/SimpleWebAuthn/blob/master/packages/browser/src/helpers/browserSupportsWebAuthnAutofill.ts#L11-L13

Nice. I like easy fixes 😊

**Comment by sameh-amwal** [comment]
Another issue worth mentioning. `isConditionalMediationAvailable()` returns true for chrome and firefox on the iPhone which both don't support however resulting in unexpected behavior for code that depends on it.

**Comment by MasterKale** [comment]
> Another issue worth mentioning. `isConditionalMediationAvailable()` returns true for chrome and firefox on the iPhone which both don't support however resulting in unexpected behavior for code that depends on it.

Hmm, there's not much I can do at the library level to account for a browser lying about its support of WebAuthn. Something like this, per-browser quirks for a given OS, would be worth documenting on a site like passkeys.dev, but as for this project, I'd rather not try to expand this to try to do OS and browser detection to guess when the API is actually available...

**Comment by sameh-amwal** [comment]
makes sense. thanks for the info about passkeys.dev. Cool resource!

**Comment by MasterKale** [comment]
@sameh-amwal This should be fixed in **@simplewebauthn/browser@6.2.2**
